chat-widget/src/components/startchaterrorpanestateful/interfaces/IStartChatErrorPaneClassNames.ts

```diff
@@ -0,0 +1,5 @@
+export interface IStartChatErrorPaneClassNames {
```

Comment:
Please update change log

Reply:
Updated to reflect `IStartChatErrorPaneProps` being added.